docker/build-package.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker build ../ --file Dockerfile-build-image -t plugin-wordpress-package-build
+docker run --mount type=bind,source="$(pwd)/..",target=/app plugin-wordpress-package-build /bin/bash -c 'cd /app && bash build.sh'
```

Comment:
```suggestion
docker run --rm --mount type=bind,source="$(pwd)/..",target=/app plugin-wordpress-package-build /bin/bash -c 'cd /app && bash build.sh'
```

no need to keep the instance